source/onboard/advanced-permissions-backend-infrastructure.rst

```diff
@@ -398,6 +410,12 @@ The following built-in roles and associated permissions are available:
 - manage_team_roles
 - sysconsole_read_compliance
 - create_emojis
+- manage_post_bleve_indexes_job
```

Comment:
@BenCookie95 - Are there any other roles these new job permissions are auto-assigned to other than system admin?

Reply:
@cwarnermm These are only assigned to the system admin